### ISSUE TITLE
Why does injecting world creation CAOS on the first launch screen attempt to create a docked world?

### ISSUE THREAD

**Issue by pushfoo** [Creatures-Developer-Network/c2e-quirks#12]
Caos run was:
```
wrld "newnameworld"
load "newnameworld"
```
Tried on lc2e from a [dstation-2020](https://github.com/Nazushvel/dstation-2020/) install using a WIP branch of [pyc2e](https://github.com/pushfoo/pyc2e) to inject.

It may be that game vars need to be set. [According to ligfx, there's a world called Startup defined in the bootstrap folder, possibly in switcher.cos](https://discord.com/channels/381123188224163841/381123347427360769/719266786452701245):
```
ligfx Today at 3:08 PM
is the world switcher environment technically a world as well?
@pushfoo yes!
pushfoo Today at 3:09 PM
interesting
so game vars in it may control world creation behavior?
ligfx Today at 3:09 PM
it's called "Startup"
yes
pushfoo Today at 3:10 PM
noted, ty. that'll be useful for minimal c2e docker images in the future
ligfx Today at 3:10 PM
DS world switcher.cos has a lot of logic for worlds
engine_no_auxiliary_bootstrap_1 is set to 1 for undocked worlds
basically, ignore C3, which is auxiliary number 1
you might be able to create a C3-only world from DS by setting engine_no_auxiliary_bootstrap_0, idk
```
This may need to be answered before larger scale dockerization is done for testing purposes.
